src/sentry/api/endpoints/organization_details.py

```diff
@@ -457,6 +458,7 @@ def save(self, *args, **kwargs):
 from rest_framework.response import Response
 
 
+@mode_limited(ServerComponentMode.CUSTOMER)
```

Comment:
@RyanSkonnord would you be able to add a test case to demonstrate that the decorator is working? 👀

Reply:
Done.